src/codegen.cpp

```diff
@@ -8272,18 +8272,18 @@ static jl_llvm_functions_t
 
     // step 7. allocate local variables slots
     // must be in the first basic block for the llvm mem2reg pass to work
-    auto allocate_local = [&](jl_varinfo_t &varinfo, jl_sym_t *s) {
+    auto allocate_local = [&ctx, &dbuilder, &debuginfo, topdebugloc, va, debug_enabled, M](jl_varinfo_t &varinfo, jl_sym_t *s, int i) {
```

Comment:
Whty the explicit captures?

Reply:
Since I noticed someone was incorrectly using `i` here, I wanted to verify the rest of the list was correct